config/locales/errors/en.yml

```diff
@@ -67,8 +67,6 @@ en:
       photo_glare: Photo has glare, please try again.
       quota_reached: Sorry your service provider has reached its identity verification
         limit.  Please contact your service provider for more information.
-      selfie: We couldn't read the photo of yourself, or match the photo to your ID.
```

Comment:
I think we do want to keep this string, maybe rebase off of https://github.com/18F/identity-idp/pull/4320